### DISCUSSION TITLE
What to do if you have lightspeed systems

### DISCUSSION THREAD

**Discussion by MilkMaker23** [3kh0/ext-remover#613] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have light speed systems so all bookmarklets are blocked so i do not know what to do

**Comment by py660** [comment]
try pressing ctrl+d, and then clicking `More...` to edit the location. this method adds the bookmark, and then lets you change it.

**Comment by py660** [comment]
However, you risk making them disappear if you mess up something

**Comment by MilkMaker23** [comment]
javascript is blocked

**Reply by py660** [reply]
you mean that if you create a bookmark linking to a NON-JAVASCRIPT LOCATION, click out of the prompt, and then EDIT the pre-existing bookmark to contain javascript code, then it will disappear?

**Reply by py660** [reply]
or is it that *ALL* bookmarks are blocked?

**Reply by Hilololsusy** [reply]
> or is it that _ALL_ bookmarks are blocked?

I think he means bookmarklets are blocked

**Reply by MilkMaker23** [reply]
@
![Screenshot 2023-04-21 12 17 14 PM](https://user-images.githubusercontent.com/96270326/233696797-9cc705db-3b40-4256-8a7b-12368e8d5b62.png)

**Reply by py660** [reply]
javascript:// is blocked, but *what about javascript:*

**Reply by py660** [reply]
eg `javascript: alert('hi')`

**Reply by MilkMaker23** [reply]
it does not work either

**Reply by py660** [reply, marked as answer]
well then, I'm out of ideas
